File: 백준/Bronze/18198. Basketball One－on－One/Basketball One－on－One.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    string st;
    cin >> st;
    vector<int> total(2, 0);
    for (int i = 0; i < st.size(); i = i + 2)
        total[st[i] - 65] += st[i + 1] - 48;
    cout << (total[0] > total[1] ? 'A' : 'B');
    return 0;
}

File: 백준/Bronze/18198. Basketball One－on－One/Basketball One－on－One.py
import sys
input=lambda:sys.stdin.readline().rstrip()
st,total=input(),[0]*2
for i in range(0,len(st),2):
    total[ord(st[i])-65]+=int(st[i+1])
print('A' if total[0]>total[1] else 'B')